### DISCUSSION TITLE
Show selected items while.. cursor is selecting them! [vid]

### DISCUSSION THREAD

**Discussion by milkoqq** [uxbox/uxbox#779] [Ideas]
Hopefully the title + video says it all.. It would be great if you could see which items are selected thus edited. 
Thank you, this is my first post ever, keep it up!

Bonus idea: Please give us the ability to change the hand-icon to cursor when in left sidebar. This oversized hyperlink hand makes everything look slow, and hyperlinked. As if everything you press will get you somewhere. 

https://user-images.githubusercontent.com/74727553/112227553-86a5c480-8c38-11eb-9513-c886ec330909.mp4

![chrome_iFmlHjQZko](https://user-images.githubusercontent.com/74727553/112228068-56125a80-8c39-11eb-8c3c-b79d2b5c851c.png)

edit: spacings

**Comment by niwinz** [comment]
Thank you @milkoqq, Looks very interesting.

I reported it internally for further discussion with the design team: https://tree.taiga.io/project/penpot/issue/1483
